content/product/product_management/okrs.md

```diff
@@ -0,0 +1,58 @@
+
+```
+CEO <-> Company OKRs <-> Executive team
+Product/Engineering VPs <-> Product/Engineering OKRs <-> Product/Engineering Directors
+Product/Engineering Directors <-> Product/Engineering org OKRs <-> Product/Engineering teams
+```
+
+### OKRs do not nest or cascade
+
+Each level of OKRs represent what is important at that level in the organization. This means that lower level OKRs do not nest under higher level OKRs, and objectives can be different at each level. Each layer of management is responsible for ensuring that the achievement of the OKRs one level below them will result in the achievement of the OKRs one level above them.
+
+Example: The VPs of Product and Engineering are responsible for ensuring that if the product and engineering OKRs are achieved, then product and engineering has done everything that needs to be done to achieve the company level OKRs.
+
+### Round-trip process
+
+The important property of OKR planning is that it is a complete round-trip with the team. If we don't do a round trip then it is possible for plans to not make business sense from a company level (if we only do bottom-up), or it is possible that the plan is infeasible (if we only do top-down). A round trip is also important because each person on the team has valuable context that we want to consider to reach the best plan.
+
+We choose to start our process top-down with a [direction document](#direction-document).
+
+Failure modes to avoid:
+
+- Orgs and teams accept direction and OKR drafts as given instead of a starting point for discussion. This failure mode is the equivalent of a top-down planning process with no round-trip.
+
+### Accounting for non-customer facing engineering work
+
+To sustain our ability to deliver over time, engineering is responsible for identifying, justifying, and prioritizing necessary technical investments (just like the product team is responsbile for identifying, justifying, and prioritizing product work). For big engineering initiatives, it may make sense to have a specific KR, but generally this kind of work can be thought of as an implementation detail of delivering on our product initiatives.
+
+This means that engineering doesn't need an explict KR to justify investment work; but they do need to be able to justify how this kind of work contributes toward our short and/or long term goals.
+
+Failure modes to avoid:
+
+- Engineering works on investments that aren't actually going to help us move faster toward short and long term goals.
+- Engineering doesn't proactively reserve time to work on necessary technical investments and our ability to deliver product improvements decreases over time.
+
+## Process
+
+### Direction document
+
+The OKR setting process starts with the VPs of Product and Engineering producing a document that describes the high level direction and strategy ([FY22-Q4 example](https://docs.google.com/document/d/1j_GuBOLXDVpzBzfCWJl-hz2rvqT--IPo3GYU07aG8HE/edit)).
```

Comment:
If what we do in that document as described in the heading https://docs.google.com/document/d/1j_GuBOLXDVpzBzfCWJl-hz2rvqT--IPo3GYU07aG8HE/edit#bookmark=id.tb35t1hjqf3z works, maybe we can bring some of those steps over here as official process.